src/components/CommandBar/CommandBarKclInput.tsx

```diff
@@ -84,17 +83,13 @@ function CommandBarKclInput({
           if (event.key === 'Backspace' && value === '') {
             event.preventDefault()
             stepBack()
+          } else if (event.key === 'Enter') {
+            event.preventDefault()
+            handleSubmit()
           }
         },
       }),
       varMentions(varMentionData),
-      EditorState.transactionFilter.of((tr) => {
-        if (tr.newDoc.lines > 1) {
-          handleSubmit()
-          return []
-        }
-        return tr
-      }),
       EditorView.updateListener.of((vu: ViewUpdate) => {
```

Comment:
I might get a review from you @franknoirot, I don't know why this transaction stopped working in the migration, but I figured listening for keydown was okay.

Reply:
Yeah I'm fairly confident that this is okay to do. This plus some issues that @iterion has been seeing with his Windows fix PR wrt the CodeMirror instance make me nervous that something is up with CodeMirror right now, so I just want to flag that.

This change looks G2G though.